aspnetcore/security/cors.md

```diff
@@ -37,27 +37,29 @@ These URLs have different origins than the previous two:
 > [!NOTE]
 > Internet Explorer doesn't consider the port when comparing origins.
 
-## Setting up CORS
+## Enable CORS
 
+::: moniker range="<= aspnetcore-1.1"
```

Comment:
We've been spacing these `moniker range` and `moniker-end` lines ... one blank line above and below.